### PR TITLE
Add Southern Company Integration Branding

### PR DESCRIPTION
**Pull request by Lash-L**
## Proposed change

Southern Company is a energy company which owns 3 major power companies in the southeast, one in Georgia, Alabama, and Mississippi. this integration aims to allow customers in those regions to get their energy data into Home Assistant.

## Type of change

- [x] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [ ] Add a new logo or icon for a custom integration (custom component)
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Removing an icon or logo

## Additional information

- This PR fixes or closes issue: fixes #
- Link to code base pull request: https://github.com/home-assistant/core/pull/87748
- Link to documentation pull request: https://github.com/home-assistant/home-assistant.io/pull/26179
- Link to integration documentation on our website: 

## Checklist

- [x] The added/replaced image(s) are **PNG**
- [x] Icon image size is 256x256px (`icon.png`)
- [x] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [x] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [x] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by Lash-L** [comment]
To be honest - I barely have enough time for the integrations I have right now, I don't need to add a third that I am responsible for. I'm going to close this issue - and if anyone comes upon it in the future, feel free to take the work I have done and I would be happy to help you get it up and running, but I don't want to be the primary code owner

If anyone wants to use this integration - you can use the HACs version of the integration: https://github.com/Lash-L/southern-company-hacs Perhaps I will reopen this branding integration later to put it in the custom component directory

**Pull request closed without merging**
